### PR TITLE
[MNG-6380] Option -Dstyle.color=always doesn't force color output

### PR DESCRIPTION
**Pull request by gnodet**

### CODE REVIEW

**Review by slachiewicz — APPROVED** [review]

**Comment by gnodet** [comment]
I need to release jansi before merging this PR.

**Review by slachiewicz — COMMENTED** [review]
I saw that something have changed in relation to unpacking libs. What native lib version will be used if we unpack (for maven core) lib to dedicated directory and set env variables also?

**Comment by michael-o** [comment]
I will have a look at this combined with other Maven issues in a few days.

**Comment by gnodet** [comment]
I've pushed commits to leverage today's changes to Jansi.

@slachiewicz I'm not sure to understand your question, could you rephrase please ?

**Comment by gnodet** [comment]
Rebased on top of #69

**Comment by gnodet** [comment]
> I saw that something have changed in relation to unpacking libs. What native lib version will be used if we unpack (for maven core) lib to dedicated directory and set env variables also?

The native libraries location have changed a bit with Jansi 2.x, so we need to properly change the maven distribution, see https://github.com/apache/maven/pull/433/commits/151e3494fec519197fb55df97e79f60bb2ef1d24

**Comment by elharo** [comment]
Running through jenkins on branch gnodet

File: pom.xml [hunk 1]
```diff
@@ -70,7 +70,7 @@
     <dependency>
       <groupId>org.fusesource.jansi</groupId>
       <artifactId>jansi</artifactId>
-      <version>2.0.1</version>
+      <version>2.2.0</version>
```
Comment (elharo) [inline pom.xml:73]: should this be 2.3.1 now?

**Review by elharo — COMMENTED** [review]
JAnsi looks released. Can this be merged?

File: pom.xml (hunk 1 above)
Comment (gnodet) [reply pom.xml:73]: We could, but the 2.3.0/2.3.1 only brings two additional native methods specifically for `mvnd` and no other changes, so there is no real benefit either.

**Comment by MartinKanters** [comment]
@gnodet I see that jansi was already upgraded in https://github.com/apache/maven-shared-utils/pull/69. Is this work still needed? Is it required or nice to have for the Maven Core 4.0.0 (alpha) release eventually?

**Comment by gnodet** [comment]
@MartinKanters jansi is a dependency both maven-shared and core maven, so it needs to be upgraded in both.  In addition, MNG-6380 is resolved with the commits in this PR and not simply the upgrade of jansi.

**Comment by gnodet** [comment]
Rebased and squashed.

**Review by slachiewicz — APPROVED** [review]

**Comment by elharo** [comment]
checking jenkins: https://ci-builds.apache.org/job/Maven/job/maven-box/job/maven-shared-utils/job/MNG-6380/

**Comment by elharo** [comment]
jenkins passed.
